pages/product-listing/product-listing.css

```diff
@@ -85,12 +117,46 @@
 	background-color: var(--primary-color);
```

Comment:
background-color applied twice

Reply:
![1](https://user-images.githubusercontent.com/89142612/154288923-4a305c3b-370b-47e5-8d44-ce5a34c7fe9b.png)
# 
Hey Ayushi, the code is collapsed here, I think that might have confused you to think it's repeating twice. If you expand the code it is not repeated.